src/msg/async/Stack.h

```diff
@@ -203,9 +203,28 @@ enum {
   l_msgr_send_messages_queue_lat,
   l_msgr_handle_ack_lat,
 
+  l_msgr_recv_encrypted_bytes_mon_cluster,
```

Comment:
I'm totally missing the use case for such fine grained reporting.  And, if there is a valid use case for it, why is it limited to encrypted traffic?

Reply:
(Covered this in a comment below).